### PR TITLE
FINAP-64/front page data binding

### PR DESCRIPTION
**Pull request by esuomi**
Real data is always nicer than fake data.

Screenshot 2022-01-25 at 14 34 09

### CODE REVIEW

File: ote/src/clj/ote/services/taxiui_service.clj [hunk 1]
```diff
@@ -104,6 +104,7 @@
 
 (defn fetch-pricing-statistics
   [db {:keys [sorting filters]}]
+  ; TODO auth checks
```
Comment (antti-nuortimo-solita) [inline ote/src/clj/ote/services/taxiui_service.clj:107]: Auth checks seem to be missing here.

Comment (antti-nuortimo-solita) [reply ote/src/clj/ote/services/taxiui_service.clj:107]: And on line 125 as well.

File: ote/src/clj/ote/services/taxiui_service.clj [hunk 2]
```diff
@@ -121,8 +122,22 @@
...
 
+(defn fetch-service-summaries
+  [db user {}]
+  [{:id "service-id-num"
+    :name "namenamename"
+    :updated "23.12.2021"
+    :example-trip 36.90
+    :price-per-kilometer 1.5
+    :price-per-minute 1
+    :operating-areas [{:label "Jokualue" :primary? true}]}]
```
Comment (antti-nuortimo-solita) [inline ote/src/clj/ote/services/taxiui_service.clj:136]: These values above seem to be some testing data and would still need to be replaced with proper values, I guess.

Comment (esuomi) [reply ote/src/clj/ote/services/taxiui_service.clj:136]: whoopsie, will remove

File: ote/src/clj/ote/services/taxiui_service.clj (hunk 1 above)
Comment (esuomi) [reply ote/src/clj/ote/services/taxiui_service.clj:107]: Funnily enough, the auth check is in the actual API endpoint definition below, just forgot to remove this TODO :) Too much coffee?

**Review by antti-nuortimo-solita — APPROVED** [review]
Looks good to me :ship:

**Pull request merged**